src/Features/Core/Portable/AddConstructorParametersFromMembers/AddConstructorParametersFromMembersCodeRefactoringProvider.AddConstructorParametersCodeAction.cs

```diff
@@ -19,32 +19,36 @@ internal partial class AddConstructorParametersFromMembersCodeRefactoringProvide
     {
         private class AddConstructorParametersCodeAction : CodeAction
         {
-            private readonly AddConstructorParametersFromMembersCodeRefactoringProvider _service;
             private readonly Document _document;
-            private readonly State _state;
-            private readonly ImmutableArray<IParameterSymbol> _missingParameters;
+            private readonly ConstructorCandidate _constructorCandidate;
+            private readonly ISymbol _containingType;
+            private readonly ImmutableArray<IParameterSymbol> _parametersToAdd;
+            private readonly bool _useSubMenuName;
```

Comment:
can you explain what thsi field is for?

Reply:
If there is more than one constructor, the suggested actions will be split into two sub menus - one for regular parameters and one for optional.  This boolean is used by the Title property to determine if the code action should be given the long title name for this:
![image](https://user-images.githubusercontent.com/25776963/54315732-36e98880-459c-11e9-95f6-dbc43d31877b.png)

or, if it should get the shorter title name for this:
![image](https://user-images.githubusercontent.com/25776963/54315780-51bbfd00-459c-11e9-8fb8-3b9d7ae7d472.png)